src/scripts/main.js

```diff
@@ -1 +1,109 @@
 'use strict';
+
+const doc = document.querySelector('body');
+
+let isFirstPromiseResolved = false;
+let isFirstPromiseRejected = false;
+
+const firstPromise = new Promise((resolve, reject) => {
+  const timeoutId = setTimeout(() => {
+    isFirstPromiseRejected = true;
+    // eslint-disable-next-line prefer-promise-reject-errors
+    reject();
```

Comment:
The `reject()` call is not passing any error or message. It's generally a good practice to provide a reason for rejection, which can help in debugging and understanding why the promise was rejected.